logs/serverDebug.log

```diff
@@ -0,0 +1,75 @@
+{"level":"info","message":"Connected to mlab","timestamp":"2017-11-14T10:35:03.996Z"}
```

Comment:
Please add this file (logs/serverDebug.log) to .gitignore and remove from repository